symbiflow.py

```diff
@@ -159,11 +159,10 @@ def run(self):
                     edam=self.edam, work_root=self.out_dir
                 )
                 self.backend.configure("")
-
             self.run_steps()
-            self.add_runtimes()
-            self.add_wirelength()
-            self.add_maximum_memory_use()
+        self.add_runtimes()
```

Comment:
Let's add a new-line between different Timed scopes, just to make it clearer (same applies to the other toolchains.